workflow-templates/im-deploy-az-swap-app-slots.yml

```diff
@@ -112,7 +112,7 @@ jobs:
       - name: Open a PagerDuty Maintenance Window
         if: env.OPEN_WINDOW == 'true'
         id: open-window
-        uses: im-open/open-pagerduty-maintenance-window@v1.0.3
+        uses: im-open/open-pagerduty-maintenance-window@v1.0.4
```

Comment:
Does this workflow template need its version incremented, too?

Reply:
yes, looks like I missed this one